pkg/sentry/fs/gofer/cache_policy.go

```diff
@@ -125,6 +125,12 @@ func (cp cachePolicy) revalidate(ctx context.Context, name string, parent, child
 		return true
 	}
 
+	// If we are not caching unstable attrs, then there is nothing to
+	// update on this inode.
+	if !cp.cacheUAttrs(child) {
```

Comment:
Doesn't this miss the case when the file has been deleted on the host, or the file has changed?  Those cases are handled by the `return true` paths on line 148 and 155